2021q2/ena.adoc

```diff
@@ -0,0 +1,32 @@
+=== ENA FreeBSD Driver Update
+
+Links: +
+link:https://github.com/amzn/amzn-drivers/blob/master/kernel/fbsd/ena/README[ENA README] URL: link:https://github.com/amzn/amzn-drivers/blob/master/kernel/fbsd/ena/README[https://github.com/amzn/amzn-drivers/blob/master/kernel/fbsd/ena/README]
+
+Contact: Michal Krawczyk <mk@semihalf.com> +
+Contact: Artur Rojek <ar@semihalf.com> +
+Contact: Marcin Wojtas <mw@semihalf.com>
+
+ENA (Elastic Network Adapter) is the smart NIC available in the virtualized environment of Amazon Web Services (AWS).
+The ENA driver supports multiple transmit and receive queues and can handle up to 100 Gb/s of network traffic, depending on the instance type on which it is used.
+
+Completed since the last update:
+
+* Update ENA driver to v2.4.0
+* Update ENA man page
+* Restructure the logging system
+* Hide sysctl nodes for unused IO queues
+* Add support for the large LLQ headers
+* Update HAL version
+
+Work in progress:
+
+* Introduce full kernel RSS API support
+* Allow reconfiguration of the RSS indirection table and hash key
+* Prototype the driver port to the iflib framework
+* MFC of the ENA v2.4.0 driver to the FreeBSD 11/12/13-STABLE branches
```

Comment:
s/MFC of/MFC/ for grammar consistency with the other bullet items, I think.

Reply:
done